synapse/config/_base.pyi

```diff
@@ -149,4 +149,6 @@ class ShardedWorkerHandlingConfig:
     instances: List[str]
     def __init__(self, instances: List[str]) -> None: ...
     def should_handle(self, instance_name: str, key: str) -> bool: ...
+
+class RoutableShardedWorkerHandlingConfig(ShardedWorkerHandlingConfig):
     def get_instance(self, key: str) -> str: ...
```

Comment:
From the above both `ShardedWorkerHandlingConfig` and `RoutableShardedWorkerHandlingConfig` seem to define `get_instance`.

Reply:
`ShardedWorkerHandlingConfig` has a private `_get_instance`, rather than `get_instance`